syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -141,6 +144,18 @@ def data(self):
         else:
             return self.native_data
 
+    @property
+    def players(self):
+        if not isinstance(self.child, ReplicatedSharingTensor):
+            raise Exception('only ReplicatedSharingTensors have property "players"')
+        return self.child.players
+
+    @property
+    def ring_size(self):
```

Comment:
Q: @abogaziah @LaRiffle what do you think about changing in the AST the name from ```field``` to ```ring_size```?

Reply:
We can leave this as it is now and we will have at one point a big round of refactoring the Tensors and then we would discuss more regarding the API

Reply:
ok